grunttasks/xssmitigate.js

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var gumbo = require('gumbo-parser');
+var sets = require('simplesets');
```

Comment:
From Travis:
```
    Loading "xssmitigate.js" tasks...ERROR
    >> Error: Cannot find module 'simplesets'
```

I think you need to save "simplesets" to package.json (and re-shrinkwrap)